### DISCUSSION TITLE
HSS .hex file for Linux AMP on reference design

### DISCUSSION THREAD

**Discussion by Alb7954** [polarfire-soc/polarfire-soc-documentation#322] [Reference designs, FPGA and development kits]
Hi,
in order to run the AMP Linux example on a reference design on Icicle-kit I should probably update the HSS in eNVM with the proper contexts definition. Where can I find the .hex to update my design ? Where can I find the .cfg for MSS configurator in order to make modification ?

Thanks

**Comment by hughbreslin** [comment]
Hey @Alb7954 

You have a few options:

1. The programming file provided for the Icicle Kit also contains the HSS .hex so programming with the job will update your fabric + eNVM. 
2. The hex file is provided as an asset with the HSS release, for example [here](https://github.com/polarfire-soc/hart-software-services/releases/tag/v2023.02.1).

The MSS configuration is provided with the reference design, you can find it [here](https://github.com/polarfire-soc/icicle-kit-reference-design/blob/master/script_support/MPFS_ICICLE_MSS_linux.cfg).
